scripts/install.sh

```diff
@@ -14,6 +14,12 @@ if [[ ${#@} -lt 1 || ${#@} -gt 2 ]]; then
     exit 1
 fi
 
+openshift_install_check=$(kubectl get deployment olm-operator -n openshift-operator-lifecycle-manager -o=jsonpath='{.spec}' 2>/dev/null)
```

Comment:
Is the string test necessary? `kubectl get` should exit with a nonzero status code if it doesn't find the resource, IIRC.